trieste/models/keras/architectures.py

```diff
@@ -220,7 +221,7 @@ def _gen_output_layer(self, input_tensor: tf.Tensor) -> tf.Tensor:
 
         distribution = dist_layer(
             self.flattened_output_shape,
-            lambda s: s.mean(),
+            tfd.Distribution.mean,
```

Comment:
that doesn't look like a formatting fix

Reply:
You're right. That was an unrelated fix. I'll revert it and check it in separately.